src/components/common/Extensions/UserHub/partials/NotificationsTab/NotificationsTab.tsx

```diff
@@ -0,0 +1,42 @@
+import { Binoculars } from '@phosphor-icons/react';
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { formatText } from '~utils/intl.ts';
+import EmptyContent from '~v5/common/EmptyContent/EmptyContent.tsx';
+
+const displayName = 'common.Extensions.UserHub.partials.NotificationsTab';
+
+const MSG = defineMessages({
+  notifications: {
+    id: `${displayName}.notifications`,
+    defaultMessage: 'Notifications',
+  },
+});
+
+const NotificationsTab = () => {
+  const isEmpty = true;
+
+  return (
+    <div className="h-full px-6 pb-6 pt-6 sm:pb-2">
+      <p className="heading-5">{formatText(MSG.notifications)}</p>
+      <div className="flex h-full flex-col justify-center pt-4 sm:h-auto sm:justify-normal">
+        {isEmpty && (
+          <>
+            <EmptyContent
+              title={{ id: 'empty.content.title.notifications' }}
```

Comment:
maybe this could be locally defined and we passed it in already i18nd via `formatText(MSG.noNotifications)` or something like that